app/workers/csv_export_worker.rb

```diff
@@ -38,7 +38,7 @@ def upload_to_s3(key, body)
       aws_secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
     )
 
-    directory = connection.directories.get(ENV['AWS_S3_BUCKET_NAME'])
+    directory = connection.directories.get(ENV['AWS_CSV_EXPORT_BUCKET_NAME'])
```

Comment:
This change, and the changes below would make a bit more sense in a different commit.

Reply:
Separated out into separate commit

Reply:
Great, thanks, I'll take a look.